packages/aws-cdk/bin/cdk.ts

```diff
@@ -255,6 +257,8 @@ async function initCommandLine() {
             trustedAccounts: args.trust,
             cloudFormationExecutionPolicies: args.cloudformationExecutionPolicies,
             terminationProtection: args.terminationProtection,
+            accessLogsBucketName: args.accessLogsBucketName,
+            accessLogsPrefix: args.accessLogsPrefix,
```

Comment:
Do we need an integration test?

[curious] How do we usually verify that this part of the code has not regressed, i.e., that passing `--access-logs-bucket-name` and prefix in the CLI end up in the bootstrap stack template?

Reply:
@nija-at can look into it, but i didn't add one for the same reason we didn't bump up the version in the`bootstrapv2` experience. the rationale there was that there's no visible change to CDK apps. You make a good point about regression.

if we were to write a test, what would you want to see:
* create logging bucket with S3 permissions
* bootstrap while providing that S3 bucket 

how much farther would we want to go in terms of verification? does it make sense to inspect the resource and verify that the bucket has access logging enabled?

I only ask this because the most recently added property was --public-access-block-configuration but the integ test just seems to check that the stack deployed and not quite the properties... that feels like it's only partially complete.

I think it makes sense to also add to the README since this parameter does require another S3 bucket, which also needs to have granted S3 permissions to deliver access logs (though ACLs). I believe deployment will fail if the bucket does not have that permission. I can write a test around that too. Passing the extra parameter will fail if the target logging bucket does not exist / have adequate permissions